templates/news-list/news-list.js

```diff
@@ -0,0 +1,50 @@
+import {
+  div, h3, p, aside, h1, a, section,
+} from '../../scripts/dom-helpers.js';
+import { getMetadata } from '../../scripts/aem.js';
+import ArticleList from '../../scripts/article-list.js';
+import { loadFragment } from '../../blocks/fragment/fragment.js';
+
+export default async function decorate(doc) {
+  const $page = doc.querySelector('main .section');
+  const $articles = div({ class: 'articles' });
+  const $pagination = div({ class: 'pagination' });
+  const articlesPerPage = Number(getMetadata('articles-per-page'));
+  const paginationMaxBtns = Number(getMetadata('pagination-max-buttons'));
+  // Old nav template
+  // const lefNavFrag = await loadFragment('/drafts/Meet/en/products/leftdiv');
+  const lefNavFrag = await loadFragment('/aside-nav');
+  // Old nav instance
+  // const $leftNav = lefNavFrag.querySelector('.accordion-wrapper').cloneNode(true);
+  const $leftNav = lefNavFrag.querySelector('.aside-nav-wrapper').cloneNode(true);
+
+  const $aside = aside(
+    $leftNav,
+  );
+
+  const $articleCard = (article) => div(
+    { class: 'card' },
```

Comment:
I would keep the attributes on the same line like so 
`div({ class: 'card'},`
 `   a({ href: article.path },`
      
it's a little easier to read.

Reply:
You can ignore eslint errors when they aren't helpful - just hover and select ignore this error for this file - it'll throw a comment at the top to tell eslint to be quiet